test/func/docker/Dockerfile

```diff
@@ -1,15 +1,26 @@
-FROM cimg/node:16.20-browsers
+FROM node:18
 
-ENV CHROME_VERSION=116
+ARG DEBIAN_FRONTEND=noninteractive
 
-USER circleci
+ARG BUILDPLATFORM
 
-COPY --chown=circleci browser-utils browser-utils
-WORKDIR browser-utils
+ENV CHROMIUM_VERSION="126.0.6478.126-1~deb12u1"
+ENV ARM_CHROMEDRIVER_URL="https://github.com/electron/electron/releases/download/v31.1.0/chromedriver-v31.1.0-linux-arm64.zip"
+ENV ARM_CHROMEDRIVER_RELATIVE_PATH=""
+ENV AMD_CHROMEDRIVER_URL="https://storage.googleapis.com/chrome-for-testing-public/126.0.6478.126/linux64/chromedriver-linux64.zip"
+ENV AMD_CHROMEDRIVER_RELATIVE_PATH="/chromedriver-linux64"
 
-RUN npm ci && npm run install-chromium
+RUN if [ "$BUILDPLATFORM" = "linux/arm64" ]; then export RELATIVE_PATH="$ARM_CHROMEDRIVER_RELATIVE_PATH"; export URL="$ARM_CHROMEDRIVER_URL"; \
+    else export RELATIVE_PATH="$AMD_CHROMEDRIVER_RELATIVE_PATH"; export URL="$AMD_CHROMEDRIVER_URL"; fi && \
+    wget "$URL" -O "chromedriver.zip" && \
+    unzip "chromedriver" && \
+    if [ "$RELATIVE_PATH" != "" ]; then mv "$RELATIVE_PATH/chromedriver" "/chromedriver"; fi && \
+    apt update -y && \
+    apt install libnss3 -y && \
+    npx playwright install chromium && \
```

Comment:
I'm not sure keeping playwright deps/browsers inside the image is a good idea. First, it makes the image much bigger. Secondly, when we'll update playwright in html-reporter, the image won't be updated and this will lead to pretty weird situation. Would it be better to install pwt deps at runtime in workflow, what do you think?